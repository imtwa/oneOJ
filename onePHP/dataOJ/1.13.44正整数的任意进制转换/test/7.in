14
20,1I4502D6078C3518F3,9
19,0B1617A4538A636,14
19,0E4081G3,12
10,675177,17
6,4500231,24
7,415036102506,2
14,1B0615A6737B,6
20,2D3476I86,15
18,3G5722A4127B76,6
17,5B8835C0324E020,12
9,200570,26
12,7A4164A2008A,9
16,6E5420B8250B11,3
10,5400708288686,6
