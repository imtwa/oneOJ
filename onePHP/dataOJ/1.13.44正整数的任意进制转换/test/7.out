273775682005582251725123
2DC6B724275566A5
167AB673A
81745
G7FJ
111110100011111000010011101001011
23222102512312521
1B83CD072B
3412431425300113542040
520512A8638603725
6JA6
22613008710327
12120211102201210222021100210211221
15253015013454142
